golang/cosmos/vm/controller.go

```diff
@@ -66,7 +65,7 @@ func UnregisterPortHandler(portNum int) error {
 func ReceiveFromController(portNum int, msg string) (string, error) {
 	handler := portToHandler[portNum]
 	if handler == nil {
-		return "", errors.New(fmt.Sprintf("Unregistered port %d", portNum))
+		return "", fmt.Errorf("Unregistered port %d", portNum)
```

Comment:
```
golang/cosmos/vm/controller.go:69:14: S1028: should use fmt.Errorf(...) instead of errors.New(fmt.Sprintf(...)) (gosimple)
		return "", errors.New(fmt.Sprintf("Unregistered port %d", portNum))
```